packages/sbcl.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Sbcl < Package
+  description 'Steel Bank Common Lisp (SBCL) is a high performance Common Lisp compiler.'
+  homepage 'http://www.sbcl.org/index.html'
+  version '1.5.3'
+  source_url 'http://prdownloads.sourceforge.net/sbcl/sbcl-1.5.3-source.tar.bz2'
+  source_sha256 'd2a7901144d227a7194661dc3ccee28873c8d5cbbd167013660c78b9ad63640c'
+
+  depends_on 'clisp' => :build
+
+  def self.build
+    system './make.sh', "--prefix=#{CREW_PREFIX}", '--xc-host=clisp --silent'
```

Comment:
Each switch option can be separated by commas like:
`... '--xc-host=clisp', '--silent'`
But this is not needed.

Reply:
@uberhacker I think he wanted the `xc-host` command to be `clisp --silent`.

Reply:
If that is the case, then that should be enclosed in quotes.  The way it works now is the switches are processed separately.